build_product_images.py

```diff
@@ -81,32 +101,41 @@ def build_image_tags(image_name, image_version, product_version):
     ]
 
 
-def build_and_publish_image(args, product):
+def build_and_publish_image(args, product) -> list[list[str]]:
```

Comment:
Python 3.8.10 returns
```
python3 ./build_product_images.py 
Traceback (most recent call last):
  File "./build_product_images.py", line 104, in <module>
    def build_and_publish_image(args, product) -> list[list[str]]:
TypeError: 'type' object is not subscriptable
```
Use together with `from typing import List`
```suggestion
def build_and_publish_image(args, product) -> List[List[str]]:
```

Reply:
Would be fine by me, works on my machine too.